app/src/main/java/com/framgia/fsalon/screen/scheduler/detail/BookingDetailActivity.java

```diff
@@ -2,8 +2,11 @@
 
 import android.content.Context;
 import android.content.Intent;
+import android.content.pm.PackageManager;
 import android.databinding.DataBindingUtil;
 import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.design.widget.Snackbar;
```

Comment:
ERROR: Unused import: android.support.design.widget.Snackbar. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports